### PR TITLE
Chore: Removing version from default user-agent header

### PR DESCRIPTION
**Pull request by yue9944882**
unless we figure out how to efficiently update the version string (`11.0.1-SNAPSHOT`) somehow..

### CODE REVIEW

**Comment by brendandburns** [comment]
Can we add a GitHub issue and a TODO so that we remember to fix this?

File: kubernetes/src/main/java/io/kubernetes/client/openapi/ApiClient.java [hunk 1]
```diff
@@ -103,7 +103,7 @@ private void init() {
     json = new JSON();
 
     // Set default User-Agent.
-    setUserAgent("Kubernetes Java Client/11.0.1-SNAPSHOT");
+    setUserAgent("Kubernetes Java Client");
```
Comment (brendandburns) [inline kubernetes/src/main/java/io/kubernetes/client/openapi/ApiClient.java:106]: Perhaps we should also add the Java runtime version?

Comment (yue9944882) [reply kubernetes/src/main/java/io/kubernetes/client/openapi/ApiClient.java:107]: added todo line

**Pull request merged**